### PR TITLE
move all alert text to partial for consistency

### PR DESCRIPTION
**Pull request by peetucket**
# Why was this change made? 🤔

Alternate approach to keep all text in the partial for consistency and pass in a key to decide which to show.

# How was this change tested? 🤨

Localhost

### CODE REVIEW

File: app/views/batch_contexts/_new_bc_form.erb [hunk 1]
```diff
@@ -43,11 +43,11 @@
                     as: :radio_buttons,
                     collection: [['Yes', true], ['No', false]] %>
                 <div data-ocr-target="runOcrImageNotes" hidden>
-                    <%= render 'alert', alert_type: 'warning', text: 'ocr_image' %>
+                    <%= render 'alert', alert_type: 'info', text: :ocr_image %>
```
Comment (peetucket) [inline app/views/batch_contexts/_new_bc_form.erb:46]: I think this should be `info` to be consistent with the document message below?

Comment (dnoneill) [reply app/views/batch_contexts/_new_bc_form.erb:46]: I am not sure I understand. I think warning is consistent. The document has two messages. A warning that says if the doc doesn't have text don't ocr it and a note (info) that says autogenerating ocr doesn't fully meet accessiblity standards.

File: app/views/batch_contexts/_new_bc_form.erb [hunk 2]
```diff
@@ -43,11 +43,11 @@
...
                 </div>
                 <div data-ocr-target="runOcrDocumentNotes" hidden>
-                    <%= render 'alert', alert_type: 'info', text: 'ocr_document' %>
-                    <%= render 'alert', alert_type: 'warning', text: 'ocr_document' %>
+                    <%= render 'alert', alert_type: 'info', text: :ocr_document %>
+                    <%= render 'alert', alert_type: 'warning', text: :ocr_accessibility %>
```
Comment (dnoneill) [inline app/views/batch_contexts/_new_bc_form.erb:50]: Line 49 should be info and ocr_accessibility
Line 50 should be warning and ocr_document

**Comment by dnoneill** [comment]
This looks good and makes sense. I think there is just one change that is needed before it can be merged into the branch. Thanks for this.

**Comment by peetucket** [comment]
Got it, fixed both problems.

**Comment by dnoneill** [comment]
I think it is still out of order. Info should come before the warning.

**Comment by peetucket** [comment]
> I think it is still out of order. Info should come before the warning.

Doh.  Ok now?

**Comment by dnoneill** [comment]
Looks good

**Pull request merged**